src/resources/js/Pages/Welcome.vue

```diff
@@ -3,30 +3,26 @@
 
         <div id="page-wrapper" class="h-screen w-full flex flex-wrap">
             <div class="bg-blue w-full text-center text-white p-1">
-                &nbsp;
+               Important Covid-19 Information. <a href="https://coronavirus.ohio.gov/wps/portal/gov/covid-19/home">Learn More.</a>
```

Comment:
They decided late in the game to not have this line in the header, so I'll take it out.